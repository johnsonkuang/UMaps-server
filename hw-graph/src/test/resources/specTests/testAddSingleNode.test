# Testing single nodes to the graph
# Tests that a single node can be added to a created graph

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1