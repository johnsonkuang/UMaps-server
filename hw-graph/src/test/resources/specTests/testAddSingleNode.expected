# Testing single nodes to the graph
# Tests that a single node can be added to a created graph

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1